*** ./smali/android/telephony/SmsMessage.smali	2016-01-18 18:51:18.742954000 +0100
--- /home/guess/patchrom/d855/temp/new_smali/telephony-common.jar.out/./smali/android/telephony/SmsMessage.smali	2016-01-18 18:52:18.902954000 +0100
***************
*** 829,834 ****
--- 829,847 ----
      return-object v0
  .end method
  
+ .method public getEncodingType()I
+     .locals 1
+ 
+     .prologue
+     iget-object v0, p0, Landroid/telephony/SmsMessage;->mWrappedSmsMessage:Lcom/android/internal/telephony/SmsMessageBase;
+ 
+     invoke-virtual {v0}, Lcom/android/internal/telephony/SmsMessageBase;->getEncodingType()I
+ 
+     move-result v0
+ 
+     return v0
+ .end method
+ 
  .method public getIndexOnIcc()I
      .locals 1
  
