*** ./smali/android/view/LayoutInflater.smali	2016-01-18 18:51:58.686954000 +0100
--- /home/guess/patchrom/d855/temp/new_smali/framework.jar.out/./smali/android/view/LayoutInflater.smali	2016-01-18 18:52:58.454954000 +0100
***************
*** 1533,1538 ****
--- 1533,1546 ----
  
      move-result-object v1
  
+     invoke-static {v1, p1}, Landroid/view/LayoutInflaterMap;->getResourceId(Landroid/content/Context;I)I
+ 
+     move-result p1
+ 
+     invoke-virtual {p0}, Landroid/view/LayoutInflater;->getContext()Landroid/content/Context;
+ 
+     move-result-object v1
+ 
      invoke-virtual {v1}, Landroid/content/Context;->getResources()Landroid/content/res/Resources;
  
      move-result-object v1
