*** ./smali/android/view/ViewDebug.smali	2016-01-18 18:51:55.966954000 +0100
--- /home/guess/patchrom/d855/temp/new_smali/framework.jar.out/./smali/android/view/ViewDebug.smali	2016-01-18 18:52:55.626954000 +0100
***************
*** 4688,4693 ****
--- 4688,4701 ----
      const-string v3, "\\n"
  
      invoke-virtual {v1, v2, v3}, Ljava/lang/String;->replace(Ljava/lang/CharSequence;Ljava/lang/CharSequence;)Ljava/lang/String;
+ 
+     move-result-object v1
+ 
+     const-string v2, "\r"
+ 
+     const-string v3, "\\r"
+ 
+     invoke-virtual {v1, v2, v3}, Ljava/lang/String;->replace(Ljava/lang/CharSequence;Ljava/lang/CharSequence;)Ljava/lang/String;
      :try_end_0
      .catchall {:try_start_0 .. :try_end_0} :catchall_0
  
