*** ./smali/android/net/IConnectivityManager.smali	2016-01-18 18:51:31.698954000 +0100
--- /home/guess/patchrom/d855/temp/new_smali/framework.jar.out/./smali/android/net/IConnectivityManager.smali	2016-01-18 18:52:32.742954000 +0100
***************
*** 447,456 ****
--- 447,472 ----
      .end annotation
  .end method
  
+ .method public abstract startUsingNetworkFeatureMSim(ILjava/lang/String;Landroid/os/IBinder;I)I
+     .annotation system Ldalvik/annotation/Throws;
+         value = {
+             Landroid/os/RemoteException;
+         }
+     .end annotation
+ .end method
+ 
  .method public abstract stopUsingNetworkFeature(ILjava/lang/String;)I
      .annotation system Ldalvik/annotation/Throws;
          value = {
              Landroid/os/RemoteException;
+         }
+     .end annotation
+ .end method
+ 
+ .method public abstract stopUsingNetworkFeatureMSim(ILjava/lang/String;I)I
+     .annotation system Ldalvik/annotation/Throws;
+         value = {
+             Landroid/os/RemoteException;
          }
      .end annotation
  .end method
