*** ./smali/android/net/wifi/WifiConfiguration$Protocol.smali	2016-01-18 18:51:31.942954000 +0100
--- /home/guess/patchrom/d855/temp/new_smali/framework.jar.out/./smali/android/net/wifi/WifiConfiguration$Protocol.smali	2016-01-18 18:52:33.002954000 +0100
***************
*** 29,35 ****
      .locals 3
  
      .prologue
!     const/4 v0, 0x2
  
      new-array v0, v0, [Ljava/lang/String;
  
--- 29,35 ----
      .locals 3
  
      .prologue
!     const/4 v0, 0x3
  
      new-array v0, v0, [Ljava/lang/String;
  
***************
*** 45,50 ****
--- 45,56 ----
  
      aput-object v2, v0, v1
  
+     const/4 v1, 0x2
+ 
+     const-string v2, "WAPI"
+ 
+     aput-object v2, v0, v1
+ 
      sput-object v0, Landroid/net/wifi/WifiConfiguration$Protocol;->strings:[Ljava/lang/String;
  
      return-void
